### PR TITLE
2850 fix browse all modal

### PR DESCRIPTION
**Pull request by evans-g-crsj**

### CODE REVIEW

File: src/components/FileRepo/AggregationSidebar/BrowseAllModal.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
+/* eslint-disable react/prop-types */
```
Comment (francisl) [inline src/components/FileRepo/AggregationSidebar/BrowseAllModal.tsx:1]: is that need in a tsx file?

**Review by francisl — COMMENTED** [review]
LGTM, one small comment

File: src/components/FileRepo/AggregationSidebar/BrowseAllModal.tsx (hunk 1 above)
Comment (evans-g-crsj) [reply src/components/FileRepo/AggregationSidebar/BrowseAllModal.tsx:1]: yes, with our configs

**Review by francisl — APPROVED** [review]
:shipit:

**Pull request merged**